src/python/grpcio_tests/tests/unit/_server_test.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2018 The gRPC Authors.
```

Comment:
Can I keep the trailing punctuation? I've always liked the trailing punctuation.

Reply:
You can, according to the regex, but I think the way I've seen most people use it, including grpc-Java, Kubernetes, and the Apache license template itself is without punctuation. It's more like a header than a sentence IMO.